.golangci.yml

```diff
@@ -1,27 +1,24 @@
 linters:
   enable:
-    - deadcode
     - errcheck
     - errorlint
     - cyclop
     - errname
     - exhaustive
     - exportloopref
     - gocritic
-    - goimports
+    - gofmt
```

Comment:
I had to replace this because it fails with some generated sources. Will see if we can revert it in a later version.